trimmer_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require './decorator'
+
+class TrimmerDecorator < Decorator
+  def correct_name
+    super[0..9]
+  end
```

Comment:
- [ ] The point of the `TrimmerDecorator` class is to trim the name only if it exceeds 10 characters, please refactor the current logic to get it right. 🙏 
- [ ] Please make sure that this class inherits from the Decorator class in which the `@namebale` instance variable and the `correct_name` method are placed, so instead of `super` it's called for to use `@namebale.correct_name`